Parsers/ASimAuditEvent/Parsers/vimAuditEventInfobloxBloxOne.yaml

```diff
@@ -0,0 +1,159 @@
+Parser:
+  Title: AuditEvent ASIM parser for Infoblox BloxOne
+  Version: '0.1.0'
+  LastUpdated: Jun 26 2024
+Product:
+  Name: Infoblox BloxOne
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM AuditEvent Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Infoblox BloxOne Documentation
+  Link: https://docs.infoblox.com/space/BloxOneThreatDefense
+Description: |
+  This ASIM parser supports normalizing AuditEvent logs from Infoblox BloxOne to the ASIM AuditEvent normalized schema. These events are captured through the Azure Monitor Agent (AMA) which are sent by the Data Connector Service of Infoblox BloxOne.
+ParserName: vimAuditEventInfbloxBloxOne
+EquivalentBuiltInParser: _Im_AuditEvent_InfobloxBloxOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: operation_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: actorusername_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: object_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: newvalue_has_any
+    Type: dynamic
+    Default: dynamic([])
+ParserQuery: |
+  let EventSeverityLookup = datatable (LogSeverity:string, EventSeverity:string)
+      [
+      "0", "Low",
+      "1", "Low",
+      "2", "Low",
+      "3", "Low",
+      "4", "Medium",
+      "5", "Medium",
+      "6", "Medium",
+      "7", "High",
+      "8", "High",
+      "9", "High",
+      "10", "High"
+  ];
+  let parser = (disabled: bool = false, starttime: datetime=datetime(null), endtime: datetime=datetime(null), eventresult: string='*', operation_has_any: dynamic=dynamic([]), eventtype_in: dynamic=dynamic([]), srcipaddr_has_any_prefix: dynamic=dynamic([]), actorusername_has_any: dynamic=dynamic([]), object_has_any: dynamic=dynamic([]), newvalue_has_any: dynamic=dynamic([])) {
+      CommonSecurityLog
+      | where not(disabled) 
+          and (isnull(starttime) or TimeGenerated >= starttime) 
+          and (isnull(endtime) or TimeGenerated <= endtime) 
+          and DeviceVendor == "Infoblox"
+          and DeviceEventClassID has "AUDIT"
+          and (eventresult == "*" or EventOutcome =~ eventresult)
```

Comment:
you can validate your changes in the automation results as shown below:
![image](https://github.com/user-attachments/assets/d381cf0a-88d1-4abb-bbee-c10383c5b533)

Reply:
you can get to validation results by clicking on ASIM validations details:
![image](https://github.com/user-attachments/assets/ff908bb7-f747-4461-9306-0f96ac8223af)

Reply:
there are 4 automated ASim tests performed for each parser file, you can view the results for each file by looking at the 'script' jobs under each test.

Reply:
We have validated the changes in the automation results and resolved the filtering errors. However, we are now encountering a new error indicating no data in the provided workspace, as shown in the image below. Could you kindly provide guidance on how to resolve this issue, @vakohl ?
![image](https://github.com/user-attachments/assets/2f4cd04a-cd44-42c4-92c8-c7ce644366e3)

Reply:
I've ingested the data. There is a bug in ingestion script. I'll be checking the result and providing my comments soon.